### PR TITLE
PR: Go to current position when using the Pdb `where` command

### PR DESCRIPTION
**Pull request by impact27**
## Description of Changes

* [ ] Wrote at least one-line docstrings (for any new functions)
* [ ] Added unit test(s) covering the changes (if testable)

* [ ] Included a screenshot or animation (if affecting the UI, see [Licecap](https://www.cockos.com/licecap/))

move to the debugging position in the editor when using the `where` command

### Issue(s) Resolved

Depends on spyder-ide/spyder-kernels#232

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct:

### CODE REVIEW

**Comment by impact27** [comment]
@ccordoba12 This one is relatively small. Do you want to include it for 4.2.0?

File: spyder/plugins/ipythonconsole/widgets/debugging.py [hunk 1]
```diff
@@ -173,6 +173,12 @@ def dbg_exec_magic(self, magic, args=''):
                     magic, args)
         self.pdb_execute(code, hidden=True)
 
+    def do_where(self):
+        """Where was called, go to the cirrent location"""
```
Comment (ccordoba12) [inline spyder/plugins/ipythonconsole/widgets/debugging.py:177]:
```suggestion
        """Where was called, go to the current location."""
```

**Comment by ccordoba12** [comment]
@impact27, please update our `spyder-kernels` subrepo to point to its `1.x` branch because I just merged spyder-ide/spyder-kernels#232.

**Review by ccordoba12 — APPROVED** [review]
Thanks @impact27!

**Pull request merged**